### PR TITLE
Release v11.0.1

### PR DESCRIPTION
**Pull request by Shaptic**

### CODE REVIEW

File: package.json
Comment (sreuland) [inline package.json:1]: does this need to bump it's [own version](https://github.com/stellar/js-stellar-base/blob/e9dc20f4f0d41adc0dc2dc59ed4d4c46a4de955e/package.json#L3) to `11.0.1` ?

Comment (Shaptic) [reply package.json:1]: I could've sworn I did that :facepalm: thank you so much, fixed

**Comment by sreuland** [comment]
is there an approach to test/verify a base version pr branch against [js-stellar-sdk](https://github.com/stellar/js-stellar-sdk/blob/master/package.json#L149) locally first, i.e. `@stellar/stellar-base: https://github.com/stellar/js-stellar-base.git#v11.0.1`, just to confirm js-stellar-sdk builds, passes?

**Review by sreuland — APPROVED** [review]
lgtm!

**Comment by Shaptic** [comment]
@sreuland good idea! I just confirmed that it worked locally with a gitref to `00cf120`.

**Pull request merged**